config_src/external/stochastic_physics/stochastic_physics.F90

```diff
@@ -0,0 +1,59 @@
+! The are stubs for ocean stochastic physics
+! the fully functional code is available at
+! http://github.com/noaa-psd/stochastic_physics
+module stochastic_physics
+
+implicit none
+
+private
+
+public :: init_stochastic_physics_ocn
+public :: run_stochastic_physics_ocn
+
+contains
+
+!!!!!!!!!!!!!!!!!!!!
+subroutine init_stochastic_physics_ocn(delt,geoLonT,geoLatT,nx,ny,nz,pert_epbl_in,do_sppt_in, &
+                                       mpiroot, mpicomm, iret)
+implicit none
```

Comment:
Even though this is just a stub bit of code to match an external interface, it would probably be a very good idea to include comments describing each of the arguments to `init_stochastic_physics_ocn()` and `run_stochastic_physics_ocn()` and the units of the real variable arguments.

Reply:
@Hallberg-NOAA do the comments to need to be doxygen style?

Reply:
We can resolve this in a later PR.  If it were serious enough to break documentation, then our testing would have caught it.